geode-core/src/main/java/org/apache/geode/cache/DiskStore.java

```diff
@@ -206,4 +207,16 @@
    * @param criticalPercent the critical percent
    */
   void setDiskUsageCriticalPercentage(float criticalPercent);
+
+  /**
+   * Returns the size unit to be used for the directories creation. Megabytes by default.
+   *
+   * @return the unit
+   */
+  DiskDirSizesUnit getDiskDirSizesUnit();
+
+  /**
+   * Sets the size unit to be used for the directories creation.
+   */
+  void setDiskDirSizesUnit(DiskDirSizesUnit unit);
```

Comment:
I think you could remove `getDiskDirSizesUnit` and `setDiskDirSizesUnit ` since they are only accessed through `DiskStoreAttributes`.